utils/rpcd-mod-attendedsysupgrade/Makefile

```diff
@@ -0,0 +1,44 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rpcd-mod-attendedsysupgrade
+PKG_VERSION:=1
+PKG_RELEASE:=1
+PKG_LICENSE:=GPL-2.0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/rpcd-mod-attendedsysupgrade
+  SECTION:=utils
+  CATEGORY:=Base system
+  TITLE:=OpenWrt ubus RPC backend server (attendedsysupgrade)
+  MAINTAINER:=Paul Spooren <paul@spooren.de>
+  DEPENDS:=rpcd +cgi-io +rpcd-mod-packagelist
+endef
+
+define Package/rpcd-mod-attendedsysupgrade/description
+	implements sysupgrade function with ubus
+	add uci settings and needed acls
```

Comment:
Is that it?  No URL with description/extended docs?    No usage guide?

Reply:
A link to https://github.com/aparcar/gsoc17-attended-sysupgrade/blob/master/README.md might make sense here. Also please use proper sentences, like "Implements a sysupgrade procedure which can be used to invoke sysupgrade via ubus calls".

Reply:
I wrote some more explanation, hopefully that explains it function good enough